curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-g.md

```diff
@@ -6,31 +6,39 @@ dashedName: introduction-flexbox-question-g
 ---
 # --description--
 
-Search through <a href="https://www.freecodecamp.org/news/the-css-flexbox-handbook" target="_blank">our handbook</a> and find the answer to the question below.
+`flex-shrink` is similar to `flex-grow`, but sets the "shrink factor" of a flex item. `flex-shrink` only ends up being applied if the size of all flex items is larger than their parent container. For example, if our 3 divs from above had a width declaration like: width: 100px, and `.flex-container` was smaller than `300px`, our divs would have to shrink to fit.
+
+The default shrink factor is `flex-shrink: 1`, which means all items will shrink evenly. If you do not want an item to shrink then you can specify `flex-shrink: 0;`. You can also specify higher numbers to make certain items shrink at a higher rate than normal.
+
+Here’s an example. If you shrink your browser window you’ll notice that `.two` never gets smaller than the given width of 250px, even though the `flex-grow` rule would otherwise specify that each element should be equally sized.
```

Comment:
```suggestion
Here's an example. If you shrink your browser window you'll notice that `.two` never gets smaller than the given width of `250px`, even though the `flex-grow` rule would otherwise specify that each element should be equally sized.
```